src/main/java/io/github/techstreet/dfscript/script/action/ScriptActionType.java

```diff
@@ -119,6 +120,31 @@ public enum ScriptActionType {
             ctx.value("Value")
         ))),
 
+    GET_REQUEST(builder -> builder.name("Get Webrequest")
+        .description("Makes a get request to the internet.")
+        .icon(Items.GRASS_BLOCK)
```

Comment:
I think something like a comparator/repeater or command block might be more fitting for a network request, if you are not going to use a head. I understand grass block as an alternative to earth but it doesn't fit "globe"?